quickstarts/atlassian/atlassian-jira/atlassian-jira.yml

```diff
@@ -0,0 +1,47 @@
+# Name of the quickstart (required)
```

Comment:
Currently, our system only supports this file being called `config.yml`. You'll need to change the name in order for this to be installable and viewable within the system.

Reply:
ahhh I knew this.. 🤣  .. good catch. fix coming in hot

Reply:
This would make a nice automated test.. it be cool to run some tests on PR submission to flag things like this before the PR is submitted.